man/docker-create.1.md

```diff
@@ -240,6 +242,22 @@ This value should always larger than **-m**, so you should always use this with
 **-w**, **--workdir**=""
    Working directory inside the container
 
+# EXAMPLES
+
+## Running a container with --interactive and without attaching to STDIN
```

Comment:
After reading this a couple more times it looks like you are trying to tell the user the right thing but the example is re-enforcing the wrong thing to do.  The heading and description should be something to the effect of 

# Never use -i without attaching STDIN